### ISSUE TITLE
I found a way to initializeApp multiple times

### ISSUE THREAD

**Issue by yaziciahmet** [firebase/firebase-admin-node#1427]
### [READ] Step 1: Are you in the right place?

  * For issues related to __the code in this repository__ file a Github issue.
  * If the issue pertains to Cloud Firestore, read the instructions in the "Firestore issue"
    template.
  * For general technical questions, post a question on [StackOverflow](http://stackoverflow.com/)
    with the firebase tag.
  * For general Firebase discussion, use the [firebase-talk](https://groups.google.com/forum/#!forum/firebase-talk)
    google group.
  * For help troubleshooting your application that does not fall under one
    of the above categories, reach out to the personalized
    [Firebase support channel](https://firebase.google.com/support/).

### [REQUIRED] Step 2: Describe your environment

  * Operating System version: _____
  * Firebase SDK version: _____
  * Firebase Product: _____ (auth, database, storage, etc)
  * Node.js version: _____
  * NPM version: _____

#### Steps to reproduce:

What happened? How can we make the problem occur?
This could be a description, log/console output, etc.

#### Relevant Code:

```
// TODO(you): code here to reproduce the problem
```

**Comment by yaziciahmet** [comment]
index.js :

const functions = require("firebase-functions");
const admin = require("firebase-admin");
const cors = require("cors")({ credentials: true, origin: true });


exports.addVehicle = functions.https.onRequest(async (req, res) => {
  cors(req, res, async () => {
    await require("./src/vehicles/addVehicle").addVehicle(req, res);
  });
});




addVehicle.js:

const admin = require("firebase-admin");
const Vehicle = require("../../models/Vehicle");

const app = admin.initializeApp();
const db = app.firestore();


exports.addVehicle = async (req, res) => {
  try{
    const vehicleInfo = new Vehicle(req.body);

    const addedVehicle = await db.collection("vehicles").add(vehicleInfo);

    res.json({data: "Succesfully added vehicle"});
  }
  catch(err){
    if(err){
      res.json(err);
    }
  }
};


Basically my question is that I use initializeApp in all of the seperate files for the functions (in addVehicle.js, getVehicles.js ex...). Is it legal or is there any downsides to this? I tried and both functions are working as expected. Normally it shouldn't be allowed to initializeApp multiple times. So I thought is it legal or is there any downsides to this? I tried and all of the functions are working as expected.
